robottelo/ui/discoveryrules.py

```diff
@@ -100,7 +100,12 @@ def update(self, name, new_name=None, search_rule=None, hostgroup=None,
         if hostgroup:
             self.select(
                 locators['discoveryrules.hostgroup_dropdown'], hostgroup)
-        self._configure_discovery(hostname, host_limit, priority, enabled)
+        self._configure_discovery(
```

Comment:
maybe it will be nice to change `create` method accordingly too:
```
         self._configure_discovery(
             hostname, host_limit, priority, locations, organizations,
             select, enabled,
         )
```